### PR TITLE
PORTAL-2090: Add an Helpdesk page about the Ask Franklin Product on the Portal helpcenter

### PR DESCRIPTION
**Pull request by manolo317**

### CODE REVIEW

File: content/md/franklin-insights/franklin-insights.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+id: franklin-insights
+themes: franklin-insights
+title: "**Connect** to Franklin Insights"
+popular: true
+related: connect-to-your-portal
+---
+
+# Overview
+
+From the **projects console**, you can access to your Franklin Insights account information and track the history of all the actions made.
```
Comment (camillebourjade) [inline content/md/franklin-insights/franklin-insights.md]:
```suggestion
From the **projects console**, you can access your Franklin Insights account information and track the history of all past actions.
```

File: content/md/franklin-insights/franklin-insights.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+**Only your Customer success manager** can activate/deactivate your account or add new credits.  
+
+# See your Franklin Insights account information
+
+1. From the menu, go to `Projects console / Projects` and select your project.
```
Comment (camillebourjade) [inline content/md/franklin-insights/franklin-insights.md]:
```suggestion
1. In the menu, go to `Projects console / Projects` and select your project.
```

File: content/md/franklin-insights/franklin-insights.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+2. Open the `Franklin Insights` tab
+3. On the `Overview` sub-tab, you can see the information about your account.
```
Comment (camillebourjade) [inline content/md/franklin-insights/franklin-insights.md]:
```suggestion
3. On the `Overview` sub-tab, you can see your account information.
```

File: content/md/franklin-insights/franklin-insights.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+The details of your **credits**, the **administrator** who is received information by email on **account activation** and **credits addition**, and your **activation code** to [use Franklin Insights in your PIM](https://help.akeneo.com/pim/v3/articles/franklin-insights-getting-started.html)
```
[the remark this answers is not in the document]
Comment (camillebourjade) [reply content/md/franklin-insights/franklin-insights.md]: We can talk about this one tomorrow morning?

File: content/md/franklin-insights/franklin-insights.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+![Franklin Insights overview tab](../img/franklin_insights_overview.jpg)
+
+# Get the log of your Franklin Insights account
+
+1. From the menu, go to `Projects console / Projects` and select your project.
```
Comment (camillebourjade) [inline content/md/franklin-insights/franklin-insights.md]:
```suggestion
1. In the menu, go to `Projects console / Projects` and select your project.
```

File: content/md/franklin-insights/franklin-insights.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+1. Open the `Franklin Insights` tab
+1. On the `Activity Log` sub-tab, you can see the history of all the actions made on your account.
```
Comment (camillebourjade) [inline content/md/franklin-insights/franklin-insights.md]:
```suggestion
1. In the `Activity Log` sub-tab, you can see the history of all the past actions in your account.
```

File: content/md/franklin-insights/franklin-insights.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (camillebourjade) [reply content/md/franklin-insights/franklin-insights.md]:
```suggestion
For example:  **credits** details, the **administrator's information**, **account activation** and **credits addition** details,  **activation code** to [use Franklin Insights](https://help.akeneo.com/pim/v3/articles/franklin-insights-getting-started.html)
```

[the remark this answers is not in the document]
Comment (camillebourjade) [reply content/md/franklin-insights/franklin-insights.md]: On the `Overview` sub-tab, you can see the information about your account: **credits** details, **administrator's information**, **account activation** and **credits addition** details,  **activation code** to [use Franklin Insights](https://help.akeneo.com/pim/v3/articles/franklin-insights-getting-started.html)

File: content/md/franklin-insights/franklin-insights.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
+---
+id: franklin-insights
+themes: franklin-insights
+title: "**Connect** to Franklin Insights"
```
Comment (Nuscly) [inline content/md/franklin-insights/franklin-insights.md]:
```suggestion
title: "Get the Franklin Insights **activation code**"
```

File: content/themes-for-customer.json [hunk 8]
```diff
@@ -27,6 +27,11 @@
             "title": "Get Helpdesk access",
             "img": "illus-permissions.svg",
             "description": "Manage the access to the Akeneo Helpdesk for your team."
+        },
+        "franklin-insights":{
+            "title": "Connect to Franklin Insights",
```
Comment (Nuscly) [inline content/themes-for-customer.json]:
```suggestion
            "title": "Get the Franklin Insights activation code",
```

File: content/themes-for-customer.json [hunk 9]
```diff
@@ -27,6 +27,11 @@
...
+            "img": "illus-franklin-robot.svg",
+            "description": "I want to access to Franklin Insights"
```
Comment (Nuscly) [inline content/themes-for-customer.json]:
```suggestion
            "description": "I want to get the Franklin Insights activation code"
```

File: content/themes-for-system-integrator.json [hunk 10]
```diff
@@ -27,6 +27,11 @@
             "title": "Get Helpdesk access",
             "img": "illus-permissions.svg",
             "description": "Manage the access to the Akeneo Helpdesk for your team."
+        },
+        "franklin-insights":{
+            "title": "Connect to Franklin Insights",
```
Comment (Nuscly) [inline content/themes-for-system-integrator.json]:
```suggestion
            "title": "Get the Franklin Insights activation code",
```

File: content/themes-for-system-integrator.json [hunk 11]
```diff
@@ -27,6 +27,11 @@
...
+            "img": "illus-franklin-robot.svg",
+            "description": "I want to access to Franklin Insights"
```
Comment (Nuscly) [inline content/themes-for-system-integrator.json]:
```suggestion
            "description": "I want to get the Franklin Insights activation code"
```

**Pull request merged**